app/src/main/java/org/mozilla/fenix/components/Analytics.kt

```diff
@@ -28,14 +29,23 @@ class Analytics(
     private val context: Context
 ) {
     val crashReporter: CrashReporter by lazy {
-        val sentryService = SentryService(
-            context,
-            BuildConfig.SENTRY_TOKEN,
-            tags = mapOf("geckoview" to "$MOZ_APP_VERSION-$MOZ_APP_BUILDID"),
-            sendEventForNativeCrashes = true
-        )
+        var services = listOf<CrashReporterService>()
+        val crashReportingEnabled = BuildConfig.CRASH_REPORTING
+        val isReleaseBuild = BuildConfig.BUILD_TYPE == "release"
+
+        if (crashReportingEnabled && isReleaseBuild) {
+            val sentryService = SentryService(
+                context,
+                BuildConfig.SENTRY_TOKEN,
+                tags = mapOf("geckoview" to "$MOZ_APP_VERSION-$MOZ_APP_BUILDID"),
+                sendEventForNativeCrashes = true
+            )
+
+            services = listOf(sentryService)
+        }
 
-        val socorroService = MozillaSocorroService(context, "Fenix")
+        val socorroService = MozillaSocorroService(context, context.getString(R.string.app_name))
+        services += socorroService
```

Comment:
We have to send at least one service to the CrashReporter or it throws ` IllegalArgumentException("No crash reporter services defined")`. That being said, no crashes should be sent as `isCrashReportingEnabled` will evaluate to false.